pkg/universe.dagger.io/x/vgjm456@qq.com/helm/install.cue

```diff
@@ -0,0 +1,58 @@
+package helm
+
+import (
+	"dagger.io/dagger"
+	"universe.dagger.io/bash"
+)
+
+#Install: {
+	// Name of your release
+	name:       string | *""
+	kubeconfig: dagger.#Secret
+	source:     *"repository" | "URL"
+	if source == "repository" {
+		chart:      string
+		reponame:   string | *"dagger_repo"
+		repository: string
+		run: {
+			env: {
+				CHART:      chart
+				REPO_NAME:  reponame
+				REPOSITORY: repository
+			}
+			script: contents: #"""
+				helm repo add $REPO_NAME $REPOSITORY
+				helm repo update
+				helm install $NAME $REPO_NAME/$CHART $GENERATE_NAME
+				"""#
+		}
+	}
+	if source == "URL" {
+		URL: string
+		run: {
+			env: "URL": URL
+			script: contents: #"""
+				helm install $NAME $URL $GENERATE_NAME
+				"""#
+		}
+	}
+
+	_base: #Image
+	run:   bash.#Run & {
+		input: _base.output
+		env: {
```

Comment:
If you use `alpine/helm` as default image, you do not need to use `bash`.
You can replace that part with that one
```
	run:   docker.#Run & {
		input: _base.output
		entrypoint: ["/bin/sh"]
		command: {
			name: "-c"
			args: [run._script]
		}
```